### PR TITLE
Multiple pages(calibredb to case) : Add Korean Translation

### PR DESCRIPTION
**Pull request by Proscream**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.ko/common/calibre-server.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# calibre-server
+
+> 네트워크를 통해 전자책을 배포하는 데 사용할 수 있는 서버 어플리케이션.
+> 이전에 GUI 또는 보정기능을 사용하여 전자책을 라이브러리로 가져와야 함.
+> Calibre 전자책 라이브러리의 일부.
+> 더 많은 정보: <https://manual.calibre-ebook.com/generated/en/calibre-server.html>.
+
+- 전자책을 배포할 서버 시작. http://localhost:8080에 연결:
```
Comment (einverne) [inline pages.ko/common/calibre-server.md]: I look up the word **연결** in the dictionary, it means connection, or link. The origin meaning here is that you can access through the link. I don't know whether it is accurate here. May your co-worker @lch7167  help to review here?

Comment (Proscream) [reply pages.ko/common/calibre-server.md]: Of course he can help this. And in Korean, almost connection is "연결", link is "링크". "링크" is English pronunciation written in Korean.

Comment (einverne) [reply pages.ko/common/calibre-server.md]: Can 연결 used as a verb?

Comment (lch7167) [reply pages.ko/common/calibre-server.md]: It looks good to me. in this situation, "연결" is short for "연결하다(verb)" :)

Comment (einverne) [reply pages.ko/common/calibre-server.md]: Ah, then make sense.

Comment (Proscream) [reply pages.ko/common/calibre-server.md]: In addition, a period usually follows the Korean verb "~하다" and was used to prevent a period.

Comment (sbrl) [reply pages.ko/common/calibre-server.md]: If @Ich7167 agrees, then we should probably leave it as-is.

**Pull request merged**